agora/shopper_insights_api/src/video_processor.py

```diff
@@ -135,6 +170,134 @@ def point_in_rectangle(self, point, rectangle):
         x, y = point
         x1, y1, x2, y2 = rectangle
         return x1 <= x <= x2 and y1 <= y <= y2
+    
+    
+
+    def create_video(self, image_folder, person_dir):
+        # Get list of image files in the folder
+        images = [
+            img for img in os.listdir(image_folder)
+            if img.lower().endswith(('.jpg', '.jpeg', '.png'))
+        ]
+
+        # Sort images based on their creation time or name
+        images.sort()
+
+        # Ensure there are enough images to process
+        if len(images) < 5:
+            print(f"Not enough images in {image_folder} to create a video. Skipping.")
+            return
+
+        # Handle resampling filter compatibility
+        try:
+            resample_filter = Image.Resampling.LANCZOS  # Pillow >= 10.0.0
+        except AttributeError:
+            resample_filter = Image.LANCZOS  # Pillow < 10.0.0
+
+        frames = []
+        desired_size = (200, 400)  # Set desired width and height
```

Comment:
why 200. 400? is this what we crop? What's the orignial video size?

Reply:
To make sure all frames have consistent dimensions, inconsistent frames causes errors when stitiching videos together